docs/UserGuide.md

```diff
@@ -88,6 +88,20 @@ Format: `time INDEX`
 Examples:
 * `time 12` returns pet 12’s pickup and dropoff times.
 
+### Adding pets’ attendance, pick-up and drop-off time: `attendance`
```

Comment:
Maybe we can make the command shorter to be more user friendly. Similar to appointment. Instead using `attendance`, we can try something like `att`.

Reply:
Do also update the command summary below